src/db/models/expenses.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const { sequelize } = require('../main');
+const { DataTypes } = require('sequelize');
+
+const Expenses = sequelize.define('Expense', {
+  userId: {
+    type: DataTypes.INTEGER,
```

Comment:
Consider adding a foreign key constraint and reference to the User model for the userId field.